content/300-guides/050-database/100-developing-with-prisma-migrate/170-customizing-migrations.mdx

```diff
@@ -65,26 +65,27 @@ To rename the `biograpy` field to `biography`:
 
 1. Edit the draft migration as shown, changing `DROP` / `DELETE` to a single `RENAME COLUMN`:
 
-      <TabbedContent tabs={[<FileWithIcon text="Before" icon="code"/>, <FileWithIcon text="After" icon="code" />]}>
```

Comment:
a lot of indentation bugs here too

Reply:
Indeed, the `TabbedContent` inside the list seems to confuse it a lot. 
Might render correctly though, need to check.